layers/+lang/javascript/README.org

```diff
@@ -149,3 +159,14 @@ You can check more [[https://github.com/mooz/js-doc/][here]]
 | Key Binding | Description                        |
 |-------------+------------------------------------|
 | ~SPC m h p~ | Get the path of the value at point |
+
+** REPL (skewer-mode)
+
+| Key Binding | Description                                                             |
+|-------------+-------------------------------------------------------------------------|
+| ~SPC m s i~ | prepares the skewer environment - run this before running REPL commands |
+| ~SPC m s r~ | starts/goes to the skewer REPL                                          |
+| ~SPC m s b~ | loads the current file in the skewer REPL                               |
+| ~SPC m s e~ | evaluates the last expression                                           |
+| ~SPC m s p~ | prints the result of the last expression                                |
+| ~SPC m s d~ | evaluates the current function at point                                |
```

Comment:
should be `SPC m s f`
see https://github.com/syl20bnr/spacemacs/blob/master/doc/CONVENTIONS.org#repls